crates/edr_napi/src/instrument.rs

```diff
@@ -4,8 +4,10 @@ use napi_derive::napi;
 
 #[napi(object)]
 pub struct InstrumentationResult {
+    /// The generated source code with coverage instrumentation.
     #[napi(readonly)]
     pub source: String,
+    /// The metadata for each piece of instrumented code.
```

Comment:
Nit: This threw me off a little because I saw `source` as a single "piece of instrumented code". This sounds a little clearer to me:

```suggestion
    /// The metadata for each instrumented code segment.
```